draft-ietf-mls-protocol.md

```diff
@@ -1000,6 +1000,70 @@ struct {
 } UserInitKey;
 ~~~~~
 
+# Message Framing
+
+[[ TODO prose ]]
+
+~~~~~
+enum {
+    invalid(0),
+    handshake(1),
+    application(2),
+    (255)
+} ContentType;
+
+struct {
+    opaque content[MLSPlaintext.length];
+    uint8 signature[MLSInnerPlaintext.sig_len];
+    uint16 sig_len;
```

Comment:
If we are gonna do this, I would prefer `opaque signature<0..2^16-1>`
otherwise if the ciphertext gives the length of the signature we don't need this.